packages/core/src/queries/application.ts

```diff
@@ -48,14 +44,19 @@ const buildConditionArray = (conditions: SqlSqlToken[]) => {
 };
 
 export const createApplicationQueries = (pool: CommonQueryMethods) => {
-  const countApplications = async (search: Search, excludeApplicationIds: string[]) => {
+  const countApplications = async (
+    search: Search,
+    excludeApplicationIds: string[],
+    types?: ApplicationType[]
```

Comment:
better to add a comment block for describing each parameter and mention conditions are joined by `and`

Reply:
Updated.